features/admin.server-configurations.v1/components/governance-connector-grid.tsx

```diff
@@ -233,6 +235,13 @@ const GovernanceConnectorCategoriesGrid: FunctionComponent<GovernanceConnectorCa
                                                     onClick={ () => handleConnectorSelection(connector) }
                                                     data-componentid={ connector.testId }
                                                 >
+                                                    {
+                                                        showStatusLabelForNewAuthzRuntimeFeatures
+                                                        && connector.status
+                                                        && (
+                                                            <div className="ribbon">{ t(connector.status) }</div>
```

Comment:
Is this intentional? Do we have a translation for this?

Reply:
Connector status is defined in `features/admin.server-configurations.v1/utils/governance-connector-utils.ts` using the `FeatureStatusLabel` enum. This PR also have translations for all the values in the defined enum.